core/helpers/EEH_Schema.helper.php

```diff
@@ -95,6 +97,12 @@ public static function add_json_linked_data_for_event(EE_Event $event)
             $template_args['venue_locality'] = $venue->city();
             $template_args['venue_region'] = $venue->state_name();
             $template_args['venue_address'] = $venue->address();
+            if ($venue->virtual_url() != '') {
```

Comment:
plz always use typesafe comparison for primitives (non-objects)

```suggestion
            if ($venue->virtual_url() !== '') {
```